docs/userguides/plugins.md

```diff
@@ -118,3 +118,36 @@ CLI arguments are decoded and passed in to the application with `click`.
 CLI plugins will use the plugin registration process defined above. The
 CLI plugins should use the `click` library in order to be able to supply
 arguments from the CLI.
+
+## Logging Ape
+
+Ape comes equiped with additional logging tools to help view output. Custom plugins may need to stop a process and log the info. The `Abort` and `ape_cli_context` are available on the `from ape.cli` and _@ape_cli_context decorator_.
+
+
+* **Abort:**
+    
+    A wrapper around a CLI exception. When you raise this error,
+    the error is nicely printed to the terminal. This is
+    useful for all user-facing errors.
+    
+    ```bash
```

Comment:
We need an example for the regular way of logging (`Abort` example can be moved.

The regular way is like this:

```python
from ape.logging import logger

logger.info("This is a log message")
```

Reply:
Do we need the Abort example if I just remove it?